scripts/test.mjs

```diff
@@ -10,5 +10,5 @@ await $`npx prisma generate --schema tests/prisma/schema.prisma`
 
 // unit tests
 console.log(chalk.blue('\nTest :: Client\n'))
```

Comment:
Does it make sense to split tests `Test :: Client`, `Test :: Server`, `Test :: Generator` or change the label to `Test :: Unit`?

Reply:
I believe tests are running in parallel by default with Vitest. So splitting them might slow down the process. But yes, feel free to change the label!